imap_processing/idex/idex_l1a.py

```diff
@@ -488,18 +484,23 @@ def _calc_high_sample_resolution(self, num_samples: int) -> npt.NDArray:
         )
         return time_high_sr_data
 
-    def _populate_bit_strings(self, packet: space_packet_parser.parser.Packet) -> None:
+    def _populate_bit_strings(
+        self, packet: space_packet_parser.packets.CCSDSPacket
+    ) -> None:
         """
         Parse IDEX data packets to populate bit strings.
 
         Parameters
         ----------
-        packet : space_packet_parser.parser.Packet
+        packet : space_packet_parser.packets.CCSDSPacket
             A single science data packet for one of the 6.
             IDEX observables.
         """
-        scitype = packet.data["IDX__SCI0TYPE"].raw_value
-        raw_science_bits = packet.data["IDX__SCI0RAW"].raw_value
+        scitype = packet["IDX__SCI0TYPE"].raw_value
+        # TODO: improve this as needed
+        raw_science_bits = "".join(
+            f"{byte:08b}" for byte in packet["IDX__SCI0RAW"].raw_value
+        )
```

Comment:
Rather than the iteration, I think we should try to do a direct conversion here:
```suggestion
        raw_science_bits = f"{int.from_bytes(packet['IDX__SCI0RAW'], byteorder='big'):0{len(packet['IDX__SCI0RAW'])*8}b}"
```

I should have mentioned this before leaving, but I think the easier change everywhere would have been to put this string conversion in the `packet_file_to_datasets()` function to keep the same output there that everyone could use. Then do all of these individual changes in another follow-up. Sorry for all the work here!

Reply:
I did think about it but I don't know if I want to include this in `packet_file_to_datasets()` function though. It will complicate things more, I think. I could create new function in `imap_processing/utils.py` that take in bytes and return binary string if we want.

Reply:
Did you want to use your new function here?

Reply:
Yes. looks like I missed this one. Good catch!